docker-hub/official_images.md

```diff
@@ -3,49 +3,33 @@ description: Guidelines for Official Images on Docker Hub
 keywords: Docker, docker, registry, accounts, plans, Dockerfile, Docker Hub, docs, official,image, documentation
 title: Docker Official Images
 redirect_from:
-- /docker-hub/official_repos/
+  - /docker-hub/official_repos/
 ---
 
 The [Docker Official Images](https://hub.docker.com/search?q=&type=image&image_filter=official){:target="_blank" rel="noopener" class="_"} are a
-curated set of Docker repositories hosted on Docker Hub. They are
+curated set of Docker repositories hosted on Docker Hub. They're
 designed to:
 
-* Provide essential base OS repositories (for example,
-  [ubuntu](https://hub.docker.com/_/ubuntu/){:target="_blank" rel="noopener" class="_"},
-  [centos](https://hub.docker.com/_/centos/){:target="_blank" rel="noopener" class="_"}) that serve as the
+- Provide essential base OS repositories (for example,
+  [Ubuntu](https://hub.docker.com/_/ubuntu/){:target="_blank" rel="noopener" class="_"},
+  [CentOS](https://hub.docker.com/_/centos/){:target="_blank" rel="noopener" class="_"}) that serve as the
```

Comment:
"While we've got the hood open" type feedback; is this a good time to swap CentOS for something else? (it's deprecated :sweat_smile:)

I'd personally love to get Debian in this list of examples, but I think Alpine is probably a less controversial addition:

```suggestion
  [Alpine](https://hub.docker.com/_/alpine/){:target="_blank" rel="noopener" class="_"}) that serve as the
```

Reply:
@tianon Oh really? That is good to know. Noted to check over any other areas of docs where this is not the case any more.